### PR TITLE
Added save_park command

### PR DESCRIPTION
**Pull request by rlewicki**
Added `save_park` command that enable user to save current state of game using command line. This pull request is related to issue #7549

### CODE REVIEW

File: src/openrct2/interface/InteractiveConsole.cpp [hunk 1]
```diff
@@ -1258,6 +1258,21 @@ static int32_t cc_for_date(
...
+{
+    if (argc < 1)
+    {
+        save_game_cmd();
+    }
+    else
+    {
+        save_game_cmd(argv[0]);
+    }
+
```
Comment (janisozaur) [inline src/openrct2/interface/InteractiveConsole.cpp]: Please remove the whitespace

File: src/openrct2/interface/InteractiveConsole.cpp [hunk 2]
```diff
@@ -1341,7 +1356,8 @@ static constexpr const console_command console_command_table[] = {
     { "remove_unused_objects", cc_remove_unused_objects, "Removes all the unused objects from the object selection.", "remove_unused_objects" },
     { "remove_park_fences", cc_remove_park_fences, "Removes all park fences from the surface", "remove_park_fences"},
     { "show_limits", cc_show_limits, "Shows the map data counts and limits.", "show_limits" },
-    { "date", cc_for_date, "Sets the date to a given date.", "Format <year>[ <month>[ <day>]]."}
+    { "date", cc_for_date, "Sets the date to a given date.", "Format <year>[ <month>[ <day>]]."},
+    { "save_park", cc_save_park, "Save current state of park. If no name specified default path will be used.", "save_park [name]"}
```
Comment (janisozaur) [inline src/openrct2/interface/InteractiveConsole.cpp]: Is a trailing comma valid here? If it is, add it

**Comment by janisozaur** [comment]
Please adhere to https://github.com/OpenRCT2/OpenRCT2/wiki/Commit-Messages, your commit message's subject line is too long.

**Comment by rlewicki** [comment]
Sorry for the commit message, didn't even realize it's that much long.

**Comment by janisozaur** [comment]
Oh and it was a frequently requested feature, it would be worth being added to changelog.

**Comment by janzev** [comment]
Does the implemented command save on the local machine or on the (headless) server?

**Comment by janisozaur** [comment]
on the local machine

**Comment by IntelOrca** [comment]
Unless the local machine is the headless server.

**Pull request merged**